seed/migrations/0125_dq_refactor.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.10 on 2020-04-13 23:03
+
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
```

Comment:
I think we may need to handle cases where a Rule had `min` and/or `max` as well as True for `required` or `not_null`. Similarly, we probably need to do something similar for cases where a Rule had `text_match` as well as True for `required` or `not_null`.

What do you envision this would look like in the new DQ world? Maybe something like: Each of those individual cases gets two new Rules created, one Rule with `condition` of "required" or "not null" and the other Rule with `condition` of "range" or "include"/"exclude".

Reply:
My understanding of the plan is on when min and max are both null, not_null or required condition applies depending on 'required' and 'not null' check box columns. If both are checked, it's 'required' ('required' must also be 'not_null', 'not_null' doesn't have to be 'required'). If a rule has a min and/or max, it'd be checking for assumably existing values which would be checking for range (for numbers, year, eui, etc.), and 'include'/'exclude' a sub string for text ('exclude' as a new condition does not exist in the database yet). That said, 'required' or 'not null' is regardless if min and/or max has/have value(s).

Reply:
Sorry for the confusion. We should revisit this soon as we can (Monday the 20th?).

Reply:
Yep - let's chat Monday. That explanation makes sense to me for the cases you presented. 

For the conversation we have on Monday. The cases that I was thinking about was when users had both min and max criteria as well as required (or not_null) criteria for one Rule. I was thinking that in this new DQ setup, we can't just ignore the min and max criteria whenever we have required or not_null criteria for a Rule being converted from the old setup.